lightning/src/chain/keysinterface.rs

```diff
@@ -206,6 +233,8 @@ pub trait ChannelKeys : Send+Clone {
 	fn commitment_seed<'a>(&'a self) -> &'a [u8; 32];
 	/// Gets the local channel public keys and basepoints
 	fn pubkeys<'a>(&'a self) -> &'a ChannelPublicKeys;
+	/// Gets the key derivation parameters in case of new derivation.
+	fn key_derivation_params(&self) -> (u64, u64);
```

Comment:
The comment should probably explain why the parameters are two integers and what their significane is.